IsraelHiking.Web/src/scss/common.scss

```diff
@@ -98,19 +98,32 @@ body {
         padding: 0 !important;
         display: block;
         width: v.$app-button-size;
-        height: v.$app-button-size;
+        height: 44px;
```

Comment:
This should use the same variable as the following definition:
https://github.com/IsraelHikingMap/Site/blob/ec5112d5aebdac923e17b597a75af6ccf609fc82/IsraelHiking.Web/src/application/components/main-menu.component.scss#L4
Or remove the definition if it is no longer needed.

Reply:
`$app-button-size` should be changed to `$app-button-width` as it only reflects the button's width.

Reply:
This variable is also used in other places. and used in both width and height.

Reply:
Then split it into two variables, it is best to use a common definition across the app for the height and width.